src/server/macos.rs

```diff
@@ -315,7 +315,22 @@ impl<'os> Method for PackageMethod<'os, Macos> {
         -> anyhow::Result<InstanceRef<'x>>
     {
         let dir = unix::storage_dir(name)?;
-        if dir.exists() {
+        let name_os = dir.file_name().ok_or_else(|| {
```

Comment:
This checks if storage dir is not root? This doesn't sound like we need it (`storage_dir` function explicitly attaches the name).

Reply:
Right, I'll `unwrap()` directly then.